.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:        
+      - v*
+
+jobs:
+  release:
+    permissions:
+      contents: read
+      id-token: write # needed for the Vault authentication
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: "Load Secrets from Vault"
+      uses: rancher-eio/read-vault-secrets@main
+      with:
+        secrets: |
+          secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
+          secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD ;
+
+    - uses: actions/checkout@v3
```

Comment:
Name is missing

Reply:
I've added a name also to the FOSSA step. :+1: